_search-plugins/cross-cluster-search.md

```diff
@@ -297,3 +303,56 @@ curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-passwo
   }
 }'
 ```
+
+## OpenSearch behind a proxy
+You can configure cross-cluster search on a cluster running behind a proxy. There are many ways to configure a reverse proxy and various proxies to choose from. The following example demonstrates the basic NGINX reverse proxy configuration without TLS termination. OpenSearch is expected to be running with both transport and HTTP TLS encryption enabled. For more information about configuring TLS encryption, see [Configuring TLS certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls/).
+
+### Proxy configuration
+The basic NGINX configuration for HTTP and transport communication follows:
```

Comment:
Where would they enter the proxy information?

Reply:
@Naarcha-AWS this would depend on how they have nginx set up, I dont think it's a good idea to specify location of this particular configuration file. Anyone who is familiar with Nginx would be able to understand this I think, But let me know if you think otherwise, I've no problem adding possible location for this conf file